.cicd/scripts/srw_ftest.sh

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
+#
+# A unified test script for the SRW application. This script is expected to
+# test the SRW application for all supported platforms.
+# The goal is to perform some basic setup and execution of the initial few workflow tasks,
+# as described in the user documentation section: "Run the Workflow Using Stand-Alone Scripts".
+# We want this following a normal SRW build in an attempt to exercise environment setup, modules,
+# data sets, and workflow scripts, without using too much time nor account resources.
+# We hope to catch any snags that might prevent WE2E fundamental testing, which follows this gate.
+# NOTE: At this time, this script is a placeholder for functional test framework.
+# At this time, we are leaving the exercise of graphical plotting for a later stage, perhaps WE2E state.
+#
+[[ -n ${ACCOUNT} ]] || ACCOUNT="no_account"
+[[ -n ${BRANCH} ]] || BRANCH="develop"
+[[ -n ${TASKS} ]] || TASKS=""
+[[ -n ${TASK_DEPTH} ]] || TASK_DEPTH=4
+[[ -n ${FORGIVE_PYTHON} ]] || FORGIVE_PYTHON=false
+set -e -u -x
+
+script_dir="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" > /dev/null 2>&1 && pwd)"
+
+# Get repository root from Jenkins WORKSPACE variable if set, otherwise, set
+# relative to script directory.
+declare workspace
+if [[ -n "${WORKSPACE}" ]]; then
+    workspace="${WORKSPACE}"
+else
+    workspace="$(cd -- "${script_dir}/../.." && pwd)"
+fi
+
+# Normalize Parallel Works cluster platform value.
+declare platform
+if [[ "${SRW_PLATFORM}" =~ ^(az|g|p)clusternoaa ]]; then
+    platform='noaacloud'
+else
+    platform="${SRW_PLATFORM}"
+fi
+
+# Test directories
+we2e_experiment_base_dir="${workspace}/expt_dirs"
+we2e_test_dir="${workspace}/tests/WE2E"
+nco_dir="${workspace}/nco_dirs"
+
+pwd
+
+# The following essentially sets up and performs tasks described in the user documentation section:
+#     "Run the Workflow Using Stand-Alone Scripts".
+echo "BRANCH=${BRANCH}"
+
+# Set the ACCOUNT to use for this PLATFORM ...
+sed "s|^  ACCOUNT: \"\"|  ACCOUNT: \"${ACCOUNT}\"|1" -i ush/config_defaults.yaml
+sed "s|hera|${platform,,}|1" ush/config.community.yaml | sed "s|an_account|${ACCOUNT}|1" > ush/config.yaml
```

Comment:
The Jenkins environment includes SRW_PROJECT, which contains the name of the account for various platforms.  I was able to make the script run by changing `${ACCOUNT}` with `${SRW_PROJECT} `on lines 51 and 52.  Additionally, you will need to add `${workspace}/` to the `ush/config.community.yaml`, `ush/config_defaults.yaml`, and `ush/config.yaml` entries on lines 51 and 52.

Reply:
regarding ACCOUNT, the intent is to run this without the need of any accounting, unless we want to run the full set of 9 tasks in the future. In that case the ACCOUNT variable here is optionally passed via the environment. For now we just want to run up to the first 4 tasks, which are relatively light, just to make sure the plumbing is in place.

Reply:
regarding 'workspace', this is set at runtime using the calling environment's WORKSPACE variable (which is defined in every jenkins job), typically the full path to the root-level of the cloned repo.   e.g.  WORKSPACE=/home/user/ufs-community/ufs-srweather-app/

Reply:
Thanks for noting that the ACCOUNT isn't needed for the first four tasks that are tested in this functional test.  I was able to run without modifying ACCOUNT to SRW_PROJECT.

Reply:
Since `ush/config*.yaml` doesn't exist in the `.cicd/scripts` subdirectory, I'm constantly getting error messages that there are no such files while attempting to manually run the script.  By adding the `${workspace}/` to these, the script is able to locate these files.  Alternatively, instead of `${workspace}/`, I suppose you could also use `../../` to point to the correct location as well.